content/docs/services/cdn-route.md

```diff
@@ -8,30 +8,57 @@ description: "Custom routes with HTTPS certificates using Amazon CloudFront"
 status: "Production Ready"
 ---
 
-Once your application is ready to move to production, you'll want a custom domain instead of the default cloud.gov one. The CDN Route service provides three key elements to support production applications: a custom domain, Content Distribution Network (CDN) caching, and TLS certificates with auto renewal.
+This service provides three key elements to support production applications:
+
+1. Custom domain support, so that your application can have your domain instead of the default `*.app.cloud.gov` domain.
+2. Content Distribution Network (CDN) caching (using [AWS CloudFront](https://aws.amazon.com/cloudfront/)), for fast delivery of content to your users.
+3. HTTPS support via free TLS certificates with auto-renewal (using [Let's Encrypt](https://letsencrypt.org/)), so that user traffic is encrypted.
 
 ## Plans
 
 Plan Name | Description | Price
 --------- | ----------- | -----
-`cdn-route` | A CDN distribution with custom URL and free TLS certificate with auto renewal | Free
+`cdn-route` | A CDN distribution with custom domain and free TLS certificate with auto renewal | Free
 
 ## Options
 
 Name | Required | Description | Default
 --- | --- | --- | ---
-`domain` | <i class="fa fa-check" aria-hidden="true"></i> | Your custom domain (or domains separated by commas) |
-`origin` | <i class="fa fa-check" aria-hidden="true"></i> | The origin root URL of the application |
-`path` |  | The path for the application within the main URL supplied | `""`
-`insecure_origin` |  | Read the application over HTTP instead of HTTPS | `false`
+`domain` | *Required* | Your custom domain (or domains separated by commas) |
+`origin` | *Optional* | *Don't put this in your command for cloud.gov tenant applications.* For services/applications that are not cloud.gov tenant applications ([more info](#external-services-and-applications)): the origin root URL of the application |
+`path` | *Optional* | The path for the application within the main domain supplied | `""`
+`insecure_origin` | *Optional* | Read the application over HTTP instead of HTTPS | `false`
+
+## How to create an instance of this service
+
+*Use these instructions for cloud.gov tenant applications. If you're creating a custom domain for something else (such as a public S3 bucket), see [external services and applications](#external-services-and-applications).*
+
+Before you begin, note that once you create a CDN service instance, you can't update or delete it until it has been successfully configured. We consider this a bug and plan to make mistakes easier to fix, but for now, consider checking your commands with a teammate to help you avoid typos and mistakes.
```

Comment:
OK, I added a commit with this info! I'm still confused about whether we need any of the old details from the custom domains page, but if you don't think we need them, I'm ok with that.